_includes/markdown/Markup.md

```diff
@@ -285,10 +285,96 @@ Combining automated and manual testing practices allows 10up to maintain a high
 <h2 id="progressive-enhancement" class="anchor-heading">Progressive Enhancement {% include Util/top %}</h2>
 Progressive enhancement means building a website that is robust, fault tolerant, and accessible. Progressive enhancement begins with a baseline experience and builds out from there, adding features for browsers that support them. It does not require us to select supported browsers or revert to table-based layouts. Baselines for browser and device support are set on a project-by-project basis.
 
-At 10up, we employ progressive enhancement to ensure that the sites we build for our clients are accessible to as many users as possible. For example, browser support for SVG has not yet reached 100%. When using SVG you should always provide a fallback such as a PNG image for browsers that do not support vector graphics.
+At 10up, we employ progressive enhancement to ensure that the sites we build for our clients are accessible to as many users as possible.
 
 ### Polyfills
 When writing markup that does not have wide browser support, using polyfills can help bring that functionality to those older browsers. Providing support for older browsers is incredibly important to the business objectives of our clients. In an effort to prevent code bloat, we only provide polyfills for features that are functionally critical to a site.
 
 ### Feature Detection
 At 10up, the concept of feature detection is used to test browser support for new features that do not yet have full support across the board. The concept of feature detection is to test if a feature is supported by the browser and if not supported, conditionality run code to provide a similar experience with browsers that do support the feature. While popular [feature detection libraries](https://modernizr.com/) exist, there are [feature detection techniques](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection#JavaScript) for JavaScript and [@supports](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports) at-rule for CSS that can be utilized.
+
+<h2 id="svg" class="anchor-heading">SVG {% include Util/top %}</h2>
+<abbr title="Scaleable Vector Graphics">SVG</abbr> has become a prevalent means for displaying rich vector graphics. At 10up we prefer to use SVGs over icon fonts for several reasons:
```

Comment:
I would like to see a touch more context for SVGs before talking about icon fonts.

Alternatively, perhaps don't _compare_ them to icon fonts. Instead outline the benefits of using SVGs without the context of icon fonts.

For example:

```
3. Semantics - icon fonts are typically rendered using pseudo-elements and an empty `<span>` element. This is not ideal semantics. Whereas, an `<svg>` is semantically an image.
```

Could become something like:

```
3. Semantics - An `<svg>` element has semantic meaning as an image. This helps communicate to browsers and assitive technology the intended meaning of the element.
```

Reply:
I created a separate section for "SVG Icons or Icon Font?" later on, and cleaned up the intro for overall SVG a bit.